modules/Bio/EnsEMBL/Analysis/Hive/Config/LastZ.pm

```diff
@@ -0,0 +1,846 @@
+  return $self->SUPER::pipeline_create_commands if $self->can('no_compara_schema');
+  my $pipeline_url = $self->pipeline_url();
+  my $parsed_url   = $second_pass && Bio::EnsEMBL::Hive::Utils::URL::parse($pipeline_url);
+  my $driver       = $second_pass ? $parsed_url->{'driver'} : '';
+
+  return [
+
+    # inheriting database and hive tables' creation
+    @{ $self->SUPER::pipeline_create_commands },
+
+    'mkdir -p ' . $self->o('compara_dump_dir'),
+    'mkdir -p ' . $self->o('compara_bed_dir'),
+
+# Compara 'release' tables will be turned from MyISAM into InnoDB on the fly by default:
+    ( $self->o('compara_innodb_schema') ? "sed 's/ENGINE=MyISAM/ENGINE=InnoDB/g' " : 'cat ' )
+      . $self->check_file_in_ensembl('ensembl-compara/sql/table.sql') . ' | ' . $self->db_cmd(),
+
+# Compara 'pipeline' tables are already InnoDB, but can be turned to MyISAM if needed:
+    ( $self->o('compara_innodb_schema') ? 'cat ' : "sed 's/ENGINE=InnoDB/ENGINE=MyISAM/g' " )
+      . $self->check_file_in_ensembl('ensembl-compara/sql/pipeline-tables.sql') . ' | ' . $self->db_cmd(),
+
+    # MySQL specific procedures
+    $driver eq 'mysql' ? ( $self->db_cmd() . ' < ' . $self->check_file_in_ensembl( 'ensembl-compara/sql/procedures.' . $driver ) ) : (),
+  ];
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{ $self->SUPER::pipeline_wide_parameters },
+    }
+}
+
+sub pipeline_analyses {
+  my ($self) = @_;
+
+  return [
+    {
+      -logic_name => 'setup_lastz',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::LastZSetup',
+      -parameters => {
+        compara_genome_db_update_path => $self->o('compara_genome_db_update_path'),
+        compara_mlss_script_path      => $self->o('compara_mlss_script_path'),
+        compara_mlss_reg_conf_path    => $self->o('compara_mlss_reg_conf_path'),
+        compara_db                    => $self->o('compara_db'),
+        projection_source_db          => $self->o('projection_source_db'),
+        target_db                     => $self->o('dna_db'),
+        pipeline_db                   => $self->o('pipeline_db'),
+        output_path                   => $self->o('output_path'),
+        compara_db_url                => 'mysql://' . $self->o('user') . ':' . $self->o('password') . '@' . $self->o('compara_db_server') . ':' . $self->o('compara_db_port') . '/' . $self->o('compara_db_name'),
+        registry_file                 => $self->o('registry_file'),
+      },
+
+      -rc_name   => '2GB_lastz',
+      -flow_into => {
+        1 => { 'get_species_list' => { 'mlss_id' => '#mlss_id#', 'reg_conf' => '#reg_conf#' } },
+      },
+    },
+
+    {
+      -logic_name => 'get_species_list',
+      -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PairAligner::ParsePairAlignerConf',
+      -parameters => {
+        'master_db'        => $self->o('compara_master'),
+        'get_species_list' => 1,
+        'core_dbs'         => undef,
+      },
+
```

Comment:
Could you please remove this blank line?

Reply:
Yep